.travis.yml

```diff
@@ -56,12 +62,23 @@ addons:
     - libatlas-base-dev
 
 before_install:
+  - SECONDS=0 && function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
IC.

Another thought: If we set it to `0`, couldn't we leave it away at all as it has not effect?